target/linux/ramips/dts/mt7620a_netcore_nw5212.dts

```diff
@@ -115,8 +126,10 @@
 };
 
 &ethernet {
-	mtd-mac-address = <&factory 0x28>;
 	mediatek,portmap = "llllw";
+
```

Comment:
Extra space? (don't hate me)

Reply:
I'm happy to see these suggestions. Thank you!